npm/pkg/dataplane/types_windows_test.go

```diff
@@ -425,7 +425,7 @@ func UpdatePolicy(policy *networkingv1.NetworkPolicy) *Action {
 
 // Do models policy updates in the NetworkPolicyController
 func (p *PolicyUpdateAction) Do(dp *DataPlane) error {
-	npmNetPol, err := translation.TranslatePolicy(p.Policy)
+	npmNetPol, err := translation.TranslatePolicy(p.Policy, false)
```

Comment:
Do we need any test cases for this set to true ?

Reply:
I think it is ok if we don't as this function is used to test windows data plane code. I have already added unit tests to test the properties within the translate function policy itself using enabled/disabled npmlite. As the primary role of this is to test data plane code which I have not changed, I believe can go with default disabled npmlite. Please let me know what you think!

Reply:
Seems ok to me to handle all testing in control plane